addons/website_event_sale/tests/test_website_event_sale.py

```diff
@@ -26,11 +26,23 @@ def test_website_event_sale(self):
             'csrf_token': http.Request.csrf_token(self),
         })
         self.assertFalse(self.env['sale.order'].search([
```

Comment:
Note that as this search depends on some implementation, we could search all SO before calling the route, then search after, and check recorsets are the same aka no so was created.